azure-cs-botservice/Azure.BotService.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi.Azure" Version="1.5.0-preview-alpha.1573228289" />
+    <PackageReference Include="Pulumi.AzureAD" Version="1.2.0-preview-alpha.1573913580" />
```

Comment:
Although versioning is a bit messy while in preview, we suggest referencing packages as `1.x.y-preview` - these are more stable versions that we produce together with each release of a provider. That would be `1.5.0-preview` and `1.2.0-preview` in this case.

Reply:
Done.